satip.c

```diff
@@ -87,7 +87,10 @@ const char *cPluginSatip::CommandLineHelp(void)
          "                                define hard-coded SAT>IP server(s)\n"
          "  -D, --detach                  set the detached mode on\n"
          "  -S, --single                  set the single model server mode on\n"
-         "  -n, --noquirks                disable all the server quirks\n";
+         "  -n, --noquirks                disable all the server quirks\n"
+         "  -r, --portrange=<start>       set a range of ports used for the rt[c]p server\n"
```

Comment:
Would it be better to define the actual port range: ```--portrange=<start>-<end>```? One could find out available ports inside the given range. Add a check that the given port range is big enough, otherwise error should be logged.